enterprise/frontend/src/metabase-enterprise/clean_up/index.tsx

```diff
@@ -17,19 +19,30 @@ if (hasPremiumFeature("collection_cleanup")) {
   PLUGIN_COLLECTIONS.useGetCleanUpMenuItems = collection => {
     const canCleanupCollection = canCleanUp(collection);
 
-    const { data: collectionItems } = useListCollectionItemsQuery(
+    const { currentData: collectionItems } = useListCollectionItemsQuery(
```

Comment:
Oh we might should do a review of our hook usages... I think we should probably be using `currentData` in a few more places.